src/smart-components/portfolio/add-portfolio-modal.js

```diff
@@ -34,7 +34,16 @@ const AddPortfolioModal = ({ removeQuery, closeTarget, viewState }) => {
 
   const onAddPortfolio = async (data) => {
     setSubmitting(true);
-    const newPortfolio = await dispatch(addPortfolio(data));
+    const notification = {
+      variant: 'success',
+      title: formatMessage(portfolioMessages.addSuccessTitle),
+      description: formatMessage(portfolioMessages.addSuccessDescription, {
+        name: data.name,
+        // eslint-disable-next-line react/display-name
+        b: (chunks) => <b>{chunks}</b>
```

Comment:
I think you should use `key` here, otherwise intl will be mad. :crab:

Reply:
Oh yeah, I hate that. Adding arbitrary keys to something we should have to :angry:

Reply:
I think i will make a wrapper around that function and fix it in all places afterwards (I've added the key here but there is a lot of places with the same issue)